src/orion/algo/mutate_functions.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.algo.mutate_functions` --
+Different mutate functions
+===========================================================================================
+
+.. module:: mutate_functions
+    :platform: Unix
+    :synopsis: Implement evolution to exploit configurations with fixed resource efficiently
+
+"""
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+def default_mutate(search_space, old_value, **kwargs):
+    """Get a default mutate function"""
+    multiply_factor = kwargs.pop('multiply_factor', 3.0)
+    add_factor = kwargs.pop('add_factor', 1)
+    volatility = kwargs.pop('volatility', 0.001)
+    lower_bound = -np.inf
+    upper_bound = np.inf
+    if search_space.type == "real":
+        if search_space.prior.name == "uniform" or \
+           search_space.prior.name == "loguniform":
```

Comment:
We should raise an error if the prior is not supported.

Reply:
see above